components/match2/wikis/halo/match_summary.lua

```diff
@@ -95,6 +97,16 @@ function CustomMatchSummary.getByMatchId(args)
 
 	match.links.lrthread = match.lrthread
 	match.links.vod = match.vod
+	if
+		--Logic.readBool(match.extradata.headtohead) and
```

Comment:
```suggestion
```

Reply:
I would like to keep this check/"switch" in the code. but commented out. In case it is determined to be needed later. If a description is needed off to the side explaining it would need added set up to be used, I can add the comment. I am unsure whether that goes against code-style though. It also will need #2243 to be merged first and added functionality for it to be used at all without error.

```suggestion
		--Logic.readBool(match.extradata.headtohead) and     The Logic check to the left will need a default added, and implementation in the HiddenDataBox(Suggested), if the Logic check would be enabled. 
```

Reply:
In general, no commented code on main. The change will always be here in the PR history, accessible easily, if it's decided its wanted